employee-rostering-frontend/src/domain/Tenant.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+interface Tenant {
+    name : string;
+    id? : number;
+    version? : number;
```

Comment:
"Wrong" indentation. By wrong I mean it violates current rules configured in `.eslintrc.json`. Feel free to use 4-space indentation, just update the rules. There are good reasons to use larger indentation. It helps readability, forces you to limit nesting of code blocks like if-then, for loops etc. On the other hand, from my experience, 2-space indentation is more common in the JavaScript world.

In any case, run `npm run lint` to check code style (and lint for code smells) and try to make it pass.